pkg/interactive/command.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Helper functions for running interactive CLI sessions from Go
+// Helper functions for running interactive docker CLI sessions from Go
```

Comment:
No, half the library is for running any commands from Go

Reply:
I've reverted the change here, but it's probably better to put these functions under different packages..